lib/octokit/client/source_import.rb

```diff
@@ -9,22 +9,34 @@ module SourceImport
       # Start a source import to a GitHub repository using GitHub Importer.
       #
       # @param repo [Integer, String, Hash, Repository] A GitHub repository.
-      # @param vcs [String] The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
       # @param vcs_url [String] The URL of the originating repository.
       # @param options [Hash]
+      # @option options [String] :vcs The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
       # @option options [String] :vcs_username If authentication is required, the username to provide to vcs_url.
       # @option options [String] :vcs_password If authentication is required, the password to provide to vcs_url.
       # @option options [String] :tfvc_project For a tfvc import, the name of the project that is being imported.
       # @return [Sawyer::Resource] Hash representing the repository import
       # @see https://developer.github.com/v3/migration/source_imports/#start-an-import
       #
       # @example
-      #   @client.import("octokit/octokit.rb", "subversion", "http://svn.mycompany.com/svn/myproject" {
+      #   @client.start_source_import("octokit/octokit.rb", "http://svn.mycompany.com/svn/myproject", {
+      #    :vcs           => "subversion",
       #    :vcs_username" => "octocat",
       #    :vcs_password  => "secret"
       #   })
-      def start_source_import(repo, vcs, vcs_url, options = {})
-        options = ensure_api_media_type(:source_imports, options.merge(:vcs => vcs, :vcs_url => vcs_url))
+      def start_source_import(repo, vcs_url, options = {}, _deprecated_args=nil)
```

Comment:
@LizzHale would you mind tackling this with a splat for consistency with the rest of the library? You can use the  `@overload` yardoc tag to communicate the different approaches. 

Here's some [prior art](https://github.com/octokit/octokit.rb/blob/719483524b930146580a997284fdd9719a5b0cdf/lib/octokit/client/commits.rb#L13-L31). I don't think your tests will change.